homeassistant/components/bond/__init__.py

```diff
@@ -25,7 +28,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
     host = entry.data[CONF_HOST]
     token = entry.data[CONF_ACCESS_TOKEN]
 
-    bond = Bond(host=host, token=token)
+    bond = Bond(host=host, token=token, timeout=ClientTimeout(total=_API_TIMEOUT))
```

Comment:
i would suggest passing this as an integer and having your module convert it to the desired object. This way if your underlaying module has to change how it handles timeouts you dont have to make changes to calls to the module.

Reply:
thanks, will apply in the follow up PR

Reply:
its likely fine eitherway, i tend to use asyncio timeout in all my modules so normally not using client timeout from aiohttp personally.

Reply:
I tried with asyncio timeout, it does work, but the error stack is much larger than when using aiohttp's timeout - which is compact and informative.

Reply:
im wondering if aiohttp timeout handling is newer or had issues in past or if most modules just accept timeout as int because theres no search results in current codebase for component using ClientTimeout. thats not to its wrong just an observation